infra/charts/turing/subcharts/mlp/db-migrations/2_applications_config.up.sql

```diff
@@ -0,0 +1,29 @@
+INSERT INTO applications(name,
+                         href,
+                         config,
+                         description,
+                         icon,
+                         use_projects,
+                         is_in_beta,
+                         is_disabled)
+VALUES ('Turing',
+        '/turing',
+        '{
+            "sections": [
+                {
+                    "name": "Routers",
+                    "href": "/routers"
+                },
+                {
+                    "name": "Ensemblers",
+                    "href": "/ensemblers"
+                },
+                {
+                    "name": "Ensembling Jobs",
+                    "href": "/jobs"
+                }
+            ]
+        }',
```

Comment:
Should we include the following in this db-migration or in a separate one as introduced in this [PR](https://github.com/gojek/mlp/pull/39)?
```
{
       "name": "Experiments",
       "href": "/experiments"
}
```

Reply:
I removed it intentionally as Turing Experiments aren't open-sourced yet. 
But moving forward, I think that populating this data (applications and applications config) by DB migrations is not a great idea (it was introduced by me so I'm fully responsible for it :D) and it should be replaced with a config file passed to MLP API at the deployment time.